labcontrol/db/process.py

```diff
@@ -3244,18 +3248,40 @@ def generate_sample_sheet(self):
     def generate_prep_information(self):
         """Generates prep information
 
+        Returns
+        -------
+        dict labcontrol.db.study.Study: str
+            a dict of the Study and the prep
+        """
+        if self.is_amplicon_assay:
+            return self._generate_amplicon_prep_information()
+        elif self.is_metagenomics_assay:
+            return self._generate_metagenomics_prep_information()
+
+        raise ValueError("Prep file generation is not implemented for this "
+                         "assay type.")
+
+    def _generate_metagenomics_prep_information(self):
+        """Generates prep information
+
         Returns
```

Comment:
I guess this is a placeholder but it doesn't return anything, right? Fine to leave as is, just noticing.